CHANGELOG.md

```diff
@@ -2,13 +2,13 @@
 
 ## Unreleased
 
-
 **Bug fixes:**
 
 - Respect country code TLDs when scrubbing span tags. ([#3458](https://github.com/getsentry/relay/pull/3458))
 
 **Features**:
 
+- **Breaking change:** Stop supporting dynamic sampling mode `"total"`, which adjusted for the client sample rate. ([#3474](https://github.com/getsentry/relay/pull/3474))
```

Comment:
Maybe it's better to add additional section for this? I think we did it in the past

Reply:
I copied this from the 24.4.0 section, but I can also make a separate section if you prefer.

Reply:
In [`23.4.0`](https://github.com/getsentry/relay/blob/master/CHANGELOG.md#2340) it was separate section, and I find it a bit better and easier to spot for out users. What do you think?

Reply:
Alright, added a new section and updated 24.4.0 section as well.